src/assets/tailwind.css

```diff
@@ -5,18 +5,48 @@
 @tailwind utilities;
 
 @layer components {
-    .base-card {
-        @apply rounded;
-    }
-    .sideBardNavigation {
-        @apply flex w-full cursor-pointer items-center rounded-l-full  pl-3 transition duration-300 ease-in-out ml-1  py-4 my-4  hover:bg-white hover:text-[#3094a1] active:scale-110; 
-
-      }
-      .sideBarNavigationActive{
-        @apply flex w-full cursor-pointer items-center rounded-l-full  pl-3 transition duration-300 ease-in-out ml-1  py-4 my-4  bg-white text-[#3094a1] active:scale-110; 
-      }
-      .sideBarButton{
-        @apply mb-auto px-1 py-3  my-auto rounded-r-full hover:text-white hover:bg-[#3094a1] border-[#3094a1] bg-white text-[#3094a1] border active:scale-110;
-      }
 
+  .base-card {
+    @apply rounded;
+  }
+  .sideBardNavigation {
+    @apply flex w-full cursor-pointer items-center rounded-l-full pl-3 transition duration-300 ease-in-out ml-1  py-4 my-4   hover:bg-background-hover hover:text-textcolor-text active:scale-110;
+  }
+  .sideBarNavigationActive {
+    @apply flex  w-full cursor-pointer items-center rounded-l-full pl-3 transition duration-300 ease-in-out ml-1  py-4 my-4   bg-white text-[#3094a1] active:scale-110;
+  }
+  .sideBarButton {
+    @apply absolute top-1/2 left-[100%] mb-auto px-1 py-5  my-auto rounded-r-full hover:text-white hover:bg-[#3094a1] border-[#3094a1] bg-white  text-[#3094a1] border active:scale-110;
```

Comment:
Här säger du att sidebarknappen ska få gå in och använda sig av maincontents space. med din absolute tag.
![absolute](https://user-images.githubusercontent.com/90420563/192383960-54f83f88-676f-4153-85c9-77f25e7a8b1b.png)

Så här ser det ut i dev just nu,. då håller sig sidebar + knapp bara inom sin flex row. och main content har sin egen flex row . 
![noAboslutPosition](https://user-images.githubusercontent.com/90420563/192383963-8efcf02d-aa42-4295-9372-0ce57ffd5415.png)